6-Typescript/server/tests/env/mockData.ts

```diff
@@ -0,0 +1,21 @@
+import { ITodo } from "../../interfaces/interfaces";
+import { v4 as uuidv4 } from "uuid";
+import chance from "chance";
+
+export const createTodo = (defParams: Partial<ITodo> = {}): ITodo => {
+    return {
+        todoText: defParams.todoText || `test${chance().integer()}`,
+        todoId: defParams.todoId || createTodoId(),
+        checked: defParams.checked || false,
+        userId: defParams.userId || createTodoId(),
+    };
+};
+export const createTodoWithMissingChecked = () => {
+    return {
+        todoText: `test${chance().integer()}`,
+        todoId: createTodoId(),
+        userId: createTodoId(),
```

Comment:
also here - `createUserId`.
you didn't catch it because both `todoId` and `userId` are strgins